LICENSE

```diff
@@ -195,6 +195,8 @@ are:
     CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
   """
 
+- flake8-i18n, licensed as "GNU General Public License v2 (GPLv2)".
```

Comment:
@charliermarsh -- I'm not certain on this but wanted to ask the question -- does including a GPL tool in Ruff mean that (all/part of) Ruff would be considered a derivative for the purposes of GPL, and therefore that Ruff would need to be re-licenced (or this PR reverted or etc)? This is the first GPL tool to be integrated, and I wanted to flag the concern as early as possible. Thanks, Adam

Reply:
Lemme look into it.

Reply:
There are already other parts including GPL-3 (flake8-django for instance).

Reply:
I'll have to get some more informed opinions on this, I'm not sure myself on whether these constitute derivative works.

If they do, and we'd have to re-license under GPL, then I'd err on the side of removing them, or going back to the authors and asking if they'd consider re-licensing under MIT or a dual license.

`flake8-django` is confusing because on PyPI it appears dual-licensed, and it includes the MIT trove classifiers:

![Screen Shot 2023-03-27 at 4 54 36 PM](https://user-images.githubusercontent.com/1309177/228064736-bfddd03b-9cee-4047-b4f4-8624716fba08.png)
![Screen Shot 2023-03-27 at 4 55 30 PM](https://user-images.githubusercontent.com/1309177/228064738-3c54e798-6c2e-437a-ac26-32fada358e54.png)

Reply:
> I'll have to get some more informed opinions on this, I'm not sure myself on whether these constitute derivative works.

Sorry to cause the trouble and thank you both for looking in to this.

> `flake8-django` is confusing because on PyPI it appears dual-licensed, and it includes the MIT trove classifiers:

It seems this has been noted already: https://github.com/rocioar/flake8-django/issues/101 and https://github.com/rocioar/flake8-django/pull/102. Sorry not to have flagged this at the time for `flake8-django`, but it seems the `MIT` trove classifier may have been a mistake when that project [adopted Poetry](https://github.com/rocioar/flake8-django/commit/bb9dcc881c36d47db932f91993015fe15c447ff3#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711).

A

Reply:
No, thank _you_ for calling this out! It's important to get right. I'll try to reach out to the `flake8-django` maintainer.